# HW_3_3_df1676


## Question: <br/><br/>Does the new bus route make a big difference for the commuters?

## Hypothesis:  

#### Notes:<br/><br/> $T_{new}: the\ commute\ time\ of\ new\ bus\ route$ <br/><br/>$T_{old}: the\ commute\ time\ of\ old\ bus\ route$<br/><br/>$T_{old} \sim N(\mu = 36, \sigma =6)$

### Null Hypothesis:<br/><br/> the commute time of new bus route is on average the same as it was before

### $$mean(T_{new}) = mean(T_{old})$$

### Alternative Hypothesis:<br/><br/>  the commute time of new bus route is on average shorter or longer than it was before 

### $$mean(T_{new})  \neq mean(T_{old})$$

### Statistical Significance: $$\alpha = 0.05$$

In [3]:
import numpy as np

In [4]:
times = np.loadtxt("times.txt")

In [6]:
N = len(times)

In [8]:
mean_pop = 36
mean_sample = times.mean()

In [9]:
stdev = 6

In [32]:
Z = (mean_pop - mean_sample) / (stdev / np.sqrt(N)) 

In [25]:
print("mean_pop = {:0.2f}, mean_sampe = {:0.2f}, stdev = {:0.2f}, N = {}".format(
        mean_pop, mean_sample, stdev, N))

mean_pop = 36.00, mean_sampe = 34.47, stdev = 6.00, N = 100


In [34]:
print ("Z = {:0.2f}".format(Z))

Z = 2.56


## Z = 2.56 means sample mean is 2.56 times stdev far from the population mean, which demonstrates H0 is more likely to reject. 

### According to the [table](http://www.sjsu.edu/faculty/gerstman/StatPrimer/z-two-tails.pdf) of Two-tail p value for Z statistics, we can find p value of our test.

p = 0.01047

### Since statistical significance in our test is <br/><br/>$\alpha = 0.05$ , $p = 0.01047 < 0.025 = \alpha/2$ , <br/><br/>then we reject Null Hypothesis and choose Alternative Hypothesis,<br/><br/> which means new bus route make a big difference for the commuters.